src/components/clickinggame/Timer.vue

```diff
@@ -0,0 +1,34 @@
+<template>
```

Comment:
Please inform me if you find a way for the parent component to be aware of the child timerCount, since I need that functionality too :)

Reply:
Yeap, I'm still working on it! :D currently my workaround is to use the `document.getElementById('timer').textContent` which I feel is not really good. Will keep you updated!